websites/B/Bluesky/presence.ts

```diff
@@ -0,0 +1,109 @@
+const presence = new Presence({
+	clientId: "1167920926076190861",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/uBFoZMP.jpg",
+		},
+		path = document.location.pathname.split("/"),
+		showUsername = await presence.getSetting<boolean>("show_profile"),
+		showPost = await presence.getSetting<boolean>("show_post"),
+		showFeed = await presence.getSetting<boolean>("show_feed");
+
+	switch (path[1]) {
+		case "profile":
+			switch (path[3]) {
+				case "feed":
+					presenceData.details = "Viewing a feed...";
+					if (showFeed) {
+						presenceData.state = document.querySelector(
+							"a.css-1jxf684.r-1loqt21:not(.r-13awgt0)"
```

Comment:
If possible, try to use selectors that don't use generated class names.

Reply:
![image](https://github.com/PreMiD/Presences/assets/38093786/eece5d1d-99e7-4784-b00c-d56b89e3c925)

It is not possible, there are only class for the name.

Reply:
You could try things like `div div > ... > a` (or similar). If that doesn't work, just know that the presence may break or need updates often.